qemu/tests/sev_basic_config.py

```diff
@@ -0,0 +1,52 @@
+from avocado.utils import process
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sev basic test on Milan and above host:
+    1. Check host sev capability
+    2. Boot sev VM
+    3. Verify sev enabled in guest
+    4. Check sev qmp cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sev test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
+    try:
+        host_check_cmd = params["sev_enable_check"].format(
+            policy_keyword="sev")
```

Comment:
it looks we need to check sev-es here?

Reply:
update to check kernel module status /sys/module/kvm_amd/parameters/sev_es